### PR TITLE
Change dotnet version of ci

### PR DESCRIPTION
**Pull request by sonohoshi**
SSIA

### CODE REVIEW

**Review by longfin — APPROVED** [review]
🙏

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -114,7 +114,7 @@ jobs:
 
       - uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: "3.1.201"
+          dotnet-version: "6.0.*"
```
Comment (moreal) [inline .github/workflows/build.yml:117]: I guess it should be `6.0.x` when looking the [example]. 🙏🏻 

[example]: https://github.com/actions/setup-dotnet

Comment (sonohoshi) [reply .github/workflows/build.yml:117]: ok I got it. thx 👍

Comment (longfin) [reply .github/workflows/build.yml:117]: `6.0.*` seems to work too 🤔  https://github.com/planetarium/NineChronicles.Headless/blob/c8ff0427cbf3d1b547eb63fc2331841eece94ef7/.github/workflows/lint.yml#L13

Comment (moreal) [reply .github/workflows/build.yml]: Oops, the both seems to work. Sorry for my mistake. 😞

Comment (sonohoshi) [reply .github/workflows/build.yml]: I think it doesn't matter it's the same result anyway. Thank you, everyone!

**Pull request merged**